packages/lesswrong/components/users/account/UsersEditForm.tsx

```diff
@@ -784,6 +784,17 @@ const UsersForm = ({
           </form.Field>
         </div>
 
+        <div className={classes.fieldWrapper}>
+          <form.Field name="notificationNewPingback">
+            {(field) => (
+              <NotificationTypeSettingsWidget
+                field={field}
+                label="Someone has mentioned my post or comment"
```

Comment:
Could you change it to this to avoid confusion with the one above:
```suggestion
                label="Someone has linked to my post or comment"
```